workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/Order.ts

```diff
@@ -15,12 +15,24 @@ export enum ORDER_CUSTOM_FIELDS {
 
 export enum ORDER_TYPES {
 	ADDONS = 'ADDONS',
-	DXPAPP = 'DXPAPP',
+	CLIENT_EXTENSION = 'CLIENT_EXTENSION',
 	CLOUDAPP = 'CLOUDAPP',
+	DXPAPP = 'DXPAPP',
+	FRAGMENT = 'FRAGMENT',
 	SOLUTIONS7 = 'SOLUTIONS7',
 	SOLUTIONS30 = 'SOLUTIONS30',
 }
 
+export const ORDER_TYPES_LABELS: {[key: string]: string} = {
+	[ORDER_TYPES.ADDONS]: 'Add-Ons',
+	[ORDER_TYPES.CLIENT_EXTENSION]: 'Client Extension',
+	[ORDER_TYPES.CLOUDAPP]: 'Cloud',
+	[ORDER_TYPES.DXPAPP]: 'DXP',
+	[ORDER_TYPES.FRAGMENT]: 'Fragment',
+	[ORDER_TYPES.SOLUTIONS7]: 'Solutions 7',
+	[ORDER_TYPES.SOLUTIONS30]: 'Solutions 30',
+};
```

Comment:
Probably the best option

```suggestion
} as const;
```